### PR TITLE
Agreement life cycle recommendations

### PR DESCRIPTION
**Pull request by maparicio**
Added recommendations for merchants on how to handle agreement life cycle. 
Added separate sub section under Step 4: Manage charges and agreements

### CODE REVIEW

File: vipps-recurring-api.md [hunk 1]
```diff
@@ -333,6 +330,23 @@ A merchant can update an agreement by calling [`PATCH:/agreements/{agreementId}`
 As a `PATCH` operation all parameters are optional. However when setting an agreement status to `STOPPED` no other changes are allowed.
 Attempts at changing other properties while stopping an agreement will result in a `400 Bad Request` response.
 
+### Agreement life cycle recommendations
+
+As seen in the section [Updating an Agreement](#Updating-an-Agreement), merchants can update agreements through our API. It's is always the merchant's responsiblity to manage and update agreements, since users are unable to make changes to their agreements directly in the Vipps app. Users will most likely request changes to their subscriptions through the merchant's channels. Here are some typical change scenarios, and corresponding reccomendations:
```
Comment (tht13) [inline vipps-recurring-api.md]: It is, not It's is

File: vipps-recurring-api.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ If it is a critical issue, or involves sensitive information please
 |:-----|:----------------------------------------------- |
 | Agreement         | A payment subscription with a set of parameters that a customer agrees to  |
 | Charge         | A single payment within an agreement |
-| Idempotency | The property of endpoints to be called multiple times without changing the result after the initial request. |
+| Idempotency | The property of endpoints to be called multiple times without changing the result beyond the initial application. |
```
Comment (tht13) [inline vipps-recurring-api.md]: Is there a reason for changing this?

Comment (maparicio) [reply vipps-recurring-api.md]: That is supposed to be the way it used to be. I'll revert it

File: vipps-recurring-api.md (hunk 1 above)
Comment (maparicio) [reply vipps-recurring-api.md]: Fixed

**Pull request merged**